project.clj

```diff
@@ -18,14 +18,15 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/certificate-authority "0.6.0"]
                  [puppetlabs/http-client "0.4.0"]
-                 [org.jruby/jruby-core "1.7.15" :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [org.jruby/jruby-core "1.7.19"
+                  :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [com.github.jnr/jffi "1.2.8"]
```

Comment:
I pinged the devs in #jruby to see if they have any info.  I agree with you that we should probably use 1.2.7 if that's what is being used in their tests, assuming we look at the 1.2.7->1.2.8 changelog and don't see any kind of huge bugfix in there that we think we need.

I'm still confused about the range thing though.  I would definitely expect the lein project snippet that you pasted above to work fine.  But if you comment those out and comment out the 'exclusion' of jffi from the core dependency, and then run lein deps tree, you should hopefully see the version range error that i'm seeing.  And I'd kind of like to understand how we're getting that range error if you think that the maven dep is pinned to 1.2.7.